srml/contract/src/tests.rs

```diff
@@ -718,7 +718,23 @@ fn deduct_blocks() {
 				* 4; // blocks to rent
 			let bob_contract = super::ContractInfoOf::<Test>::get(BOB).unwrap().get_alive().unwrap();
 			assert_eq!(bob_contract.rent_allowance, 1_000 - rent);
+			assert_eq!(bob_contract.deduct_block, 5);
 			assert_eq!(Balances::free_balance(BOB), 30_000 - rent);
+
+			// Advance 7 blocks more
```

Comment:
```suggestion
			// Advance 12 blocks more
```

Reply:
well either advance to block 12 or advance 7 blocks more. But not 12 blocks more